packages/components/src/List/utils/getListItemDimensions.ts

```diff
@@ -0,0 +1,51 @@
+/*
+
+ MIT License
+
+ Copyright (c) 2020 Looker Data Sciences, Inc.
+
+ Permission is hereby granted, free of charge, to any person obtaining a copy
+ of this software and associated documentation files (the "Software"), to deal
+ in the Software without restriction, including without limitation the rights
+ to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ copies of the Software, and to permit persons to whom the Software is
+ furnished to do so, subject to the following conditions:
+
+ The above copyright notice and this permission notice shall be included in all
+ copies or substantial portions of the Software.
+
+ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ SOFTWARE.
+
+ */
+
+import { DensityRamp, ListItemDimensions } from '../types'
+import {
+  density0,
+  densityP1,
+  densityN1,
+  densityN2,
+  densityN3,
+} from './dimensions'
+
+export const getListItemDimensions = (
+  density: DensityRamp
+): ListItemDimensions => {
+  switch (density) {
+    case -1:
+      return { ...densityN1 }
+    case -2:
+      return { ...densityN2 }
+    case -3:
+      return { ...densityN3 }
+    case 1:
+      return { ...densityP1 }
+    default:
+      return { ...density0 }
+  }
+}
```

Comment:
With the object above this could become a simple object lookup:


```suggestion
export const getListItemDimensions = (
  density: DensityRamp
): ListItemDimensions => densities[density]
```

Reply:
That works for me! Honestly, my biggest thing with the object route was the naming convention I whipped up, but if it's passable there's definitely no downside afaik to having these available.

Reply:
Resolved in ab951f7